modules/troubleshooting/pages/system-logs.adoc

```diff
@@ -1,10 +1,11 @@
-= Log files
+= System Logs
 
 TigerGraph Database captures key information on activities occurring across its different components through log functions that output to log files. These log files are not only helpful in xref:troubleshooting-guide.adoc[troubleshooting] but also serve as an auditory resource. This document gives a high-level overview of TigerGraph's logging structure and lists some common information one might need to monitor their database services and where to obtain them in the logs.
```

Comment:
maybe it would make sense to put the information on this page together with the log overview page?